src/workflow/ParameterManager.py

```diff
@@ -24,6 +24,32 @@ def __init__(self, workflow_dir: Path):
         self.params_file = Path(workflow_dir, "params.json")
         self.param_prefix = f"{workflow_dir.stem}-param-"
         self.topp_param_prefix = f"{workflow_dir.stem}-TOPP-"
+        
+        
+    
+    def get_boolean_params(self, tool: str) -> list:
+        ini_file = Path(self.ini_dir, f"{tool}.ini")
+        if not ini_file.exists():
+            return []
+
+        param = poms.Param()
+        poms.ParamXMLFile().load(str(ini_file), param)
+
+        boolean_params = []
+
+        for key in param.keys():
+            try:
+                entry = param.getEntry(key)
+                # pyopenms 3.3.0 uses valueType == 3 for boolean
+                if hasattr(entry, 'valueType') and entry.valueType == 3:
```

Comment:
The underlying enum looks like this:

```
enum ValueType : unsigned char
    {
      STRING_VALUE, ///< string value
      INT_VALUE, ///< integer value
      DOUBLE_VALUE, ///< double value
      STRING_LIST, ///< string vector
      INT_LIST, ///< integer vector
      DOUBLE_LIST, ///< double vector
      EMPTY_VALUE ///< empty value
    };
```
Unfortunately - both boolean and strings get mapped to `STRING_VALUE` and cannot be distinguished by their value type attribute. Thus, I think we will have to inspect the ini to distinguish this.